pytests/tests/test_baseurls.py

```diff
@@ -32,11 +32,37 @@ def teardown_test(utils):
         os.remove(filename)
 
 
-def test_multiple_baseurls(utils):
+def test_multiple_repoid(utils):
     reponame = TESTREPO
     workdir = WORKDIR
     utils.makedirs(workdir)
 
+    filename = os.path.join(utils.config['repo_path'], "yum.repos.d", REPOFILENAME)
+
+    baseurls = "http://localhost:8080/photon-test"
+    utils.create_repoconf(filename, baseurls, reponame)
+
+    ret = utils.run(['tdnf',
+                     '--disablerepo=*', '--enablerepo={}'.format(reponame),
+                     'makecache'],
+                    cwd=workdir)
+    assert ret['retval'] == 0
```

Comment:
If we have multipe repos with the same id, `makecache` should already throw an error, so you should assert for `1037` here.

Reply:
Done